templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+    <h1>{{ object|yesno:"Update,Create" }} Manufactured</h1>
```

Comment:
The heading 'Update,Create Manufactured' seems to have a typo. It should be 'Update,Create Manufacturer' to correctly reflect the entity being managed.